### PR TITLE
fix: OPTIC-669: ResizeObserver errors in sentry

### PR DESCRIPTION
**Pull request by bmartel**
### PR fulfills these requirements
- [X] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [X] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [X] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [X] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [X] Frontend

### Describe the reason for change
There are a flood of errors coming from the usages of ResizeObserver, some are possibly legitimate and point to a defect in the code logic, others are possibly from conflict within third-party libraries, or browser extension influencing the errors through UI manipulation that is not expected.

This PR is looking to surface these errors through our headful integration tests, and hopefully find some resolutions to these so we can quell the influx of noise.

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [X] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [x] e2e
- [x] integration
- [ ] unit

### CODE REVIEW

File: web/libs/editor/src/components/VideoCanvas/VideoCanvas.tsx [hunk 1]
```diff
@@ -342,7 +342,9 @@ export const VideoCanvas = memo(
 
     useEffect(() => {
       const observer = new ResizeObserver(() => {
-        props.onResize?.(videoDimensions);
+        requestAnimationFrame(() => {
+          props.onResize?.(videoDimensions);
+        })
```
Comment (bmartel) [inline web/libs/editor/src/components/VideoCanvas/VideoCanvas.tsx:347]:
```suggestion
        props.onResize?.(videoDimensions);
```

Comment (bmartel) [reply web/libs/editor/src/components/VideoCanvas/VideoCanvas.tsx:347]: Putting back to see if this passes tests without failures.

File: web/libs/editor/tests/e2e/codecept.conf.js [hunk 2]
```diff
@@ -111,7 +111,7 @@ module.exports.config = {
       uncaughtErrorFilter: {
         interrupt: true,
         ignore: [
-          /^ResizeObserver loop limit exceeded$/,
+          // /^ResizeObserver loop limit exceeded$/,
```
Comment (bmartel) [inline web/libs/editor/tests/e2e/codecept.conf.js]:
```suggestion
```

File: web/libs/editor/tests/e2e/fragments/ErrorsCollector.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ function startErrorsCollector() {
 
   CEErrorsCollector.prototype.errorHandler = function (ev) {
     // Ignore not meaningful error
-    if (ev.message === "ResizeObserver loop limit exceeded") return;
+    // if (ev.message === "ResizeObserver loop limit exceeded") return;
```
Comment (bmartel) [inline web/libs/editor/tests/e2e/fragments/ErrorsCollector.js]:
```suggestion
```

File: web/libs/editor/tests/integration/support/e2e.ts [hunk 4]
```diff
@@ -3,13 +3,13 @@ import "@humansignal/frontend-test/cypress/support/e2e";
...
+  return err.message.includes("ResizeObserver");
 });
 
 beforeEach(() => {
-  cy.on("uncaught:exception", (err) => {
-    return !err.message.includes("ResizeObserver loop completed with undelivered notifications.");
-  });
+  // cy.on("uncaught:exception", (err) => {
+  //   return !err.message.includes("ResizeObserver loop completed with undelivered notifications.");
+  // });
```
Comment (bmartel) [inline web/libs/editor/tests/integration/support/e2e.ts]:
```suggestion
```

File: web/libs/storybook/tests/support/component.ts [hunk 5]
```diff
@@ -1,7 +1,9 @@
 import { mount } from "cypress/react";
 
 Cypress.on("uncaught:exception", (err) => {
-  const ignoredErrors = ["ResizeObserver loop completed with undelivered notifications."];
+  const ignoredErrors: Array<string> = [
+    // "ResizeObserver loop completed with undelivered notifications."
```
Comment (bmartel) [inline web/libs/storybook/tests/support/component.ts]:
```suggestion
```

**Review by Gondragos — APPROVED** [review]

**Review by luarmr — APPROVED** [review]

**Pull request merged**